### PR TITLE
Add links to Why we made this guide, fix footer copy, fix email error…

### PR DESCRIPTION
**Pull request by TheoChevalier**
… copy

### CODE REVIEW

**Comment by TheoChevalier** [comment]
Cc @ScottDowne

**Comment by TheoChevalier** [comment]
Adding the category names

File: src/components/buyers-guide/signup-form.js [hunk 1]
```diff
@@ -63,7 +62,7 @@ var SignupForm = React.createClass({
     if (!this.state.emailInput.trim()) {
       valid = false;
 
-      emailError = this.context.intl.formatMessage({id: 'required_error'});
+      emailError = this.context.intl.formatMessage({id: 'email_invalid'});
```
[the remark this answers is not in the document]
Comment (ScottDowne) [reply src/components/buyers-guide/signup-form.js]: Other than this, I think this looks good to me.

[the remark this answers is not in the document]
Comment (TheoChevalier) [reply src/components/buyers-guide/signup-form.js]: It says “Oops! You forgot to click this little box.” under the email field

[the remark this answers is not in the document]
Comment (TheoChevalier) [reply src/components/buyers-guide/signup-form.js]: Maybe we can add back the old copy and add a third string just for this case, and say “This filed is required”

[the remark this answers is not in the document]
Comment (ScottDowne) [reply src/components/buyers-guide/signup-form.js]: Ah, I see what happened. (I wasn't aware (or forgot) we changed the error copy) but yeah this makes sense to me now 👍

[the remark this answers is not in the document]
Comment (TheoChevalier) [reply src/components/buyers-guide/signup-form.js]: Ok, added the old copy back

**Pull request merged**